src/main/groovy/com/ullink/NuGetExtension.groovy

```diff
@@ -2,4 +2,5 @@ package com.ullink
 
 class NuGetExtension {
     String version = '3.4.3'
+    String nugetUrl = null
```

Comment:
I'd put this by default to `https://dist.nuget.org/win-x86-commandline/v$version/nuget.exe`.

Then we don't need all the extra logic inside the BaseNuGet.groovy.